### PR TITLE
Add migration system

### PR DESCRIPTION
**Pull request by g1y**
Adds an actual migration system so that migrations do not have to be run by hand.

Only adds the dependency and an example config file.

### CODE REVIEW

File: database.example.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (danielj41) [inline database.example.js]: I think this should be `database.example.json`: 

> Location of the database.json file.             [default: "./database.json"]

from https://db-migrate.readthedocs.io/en/latest/Getting%20Started/installation/

**Comment by danielj41** [comment]
I think it'd be useful to include a sample migration that doesn't do anything. If nothing else, it'd help QA this pull.

**Comment by danielj41** [comment]
CR 👍  but dev_block 👍  on the `json` config file.

File: database.example.js (hunk 1 above)
Comment (MooreGuy) [reply database.example.js]: :+1: Ah, yes, thank you.

**Comment by MooreGuy** [comment]
Yeah, I that's probably a good idea, to show that it works, I'll think of a migration to add.

**Comment by danielbeardsley** [comment]
> I'll think of a migration to add.

You could make a no-op migration to show how it works.

I read through all the docs of `db-migrate` and while it sounds like a cool tool, they don't really explain how to go about start using it with an existing project. i.e. should we use our entire schema.sql file as the first migration?

I just don't see us having enough migrations to make this worthwhile, but maybe I'm not thinking far enough into the future?

**Comment by sterlinghirsh** [comment]
Status update? This has been blocked for months.

**Comment by g1y** [comment]
@copperwall was looking into a nice RDBMS, but I 'm not sure if he'll be going forward with that. If he did decide he wanted to use that, this would definitely conflict with that. I want his comment on what he wants to do before going forward with this.

Otherwise, I'm sure people would love to have this.

**Comment by copperwall** [comment]
I still want to do that D: We can talk about it more when I get back next week

**Pull request closed without merging**